recipes-browser/webkitgtk/webkitgtk_2.30.5.bb

```diff
@@ -25,12 +25,7 @@ RRECOMMENDS_${PN}-bin = "adwaita-icon-theme librsvg-gtk"
 
 inherit cmake lib_package pkgconfig perlnative python3native
 
-# distro_features_check is going to be removed after dunfell
-# ref: https://git.yoctoproject.org/cgit/cgit.cgi/poky/commit/meta/classes/features_check.bbclass?h=master-next&id=9702544b3e75d761d86cae7e8b36f3f2625b68ce
-#   Temporarily support the old class name with a warning about future deprecation.
-inherit ${@bb.utils.contains_any("LAYERSERIES_CORENAMES", 'zeus warrior', 'distro_features_check', 'features_check', d)}
```

Comment:
Why removing this?
Isn't it needed to use the variable ```DISTRO_FEATURES``` ?

Reply:
No it is needed for `REQUIRED_DISTRO_FEATURES`, `ANY_OF_DISTRO_FEATURES` and `CONFLICT_DISTRO_FEATURES`

Reply:
Ok